torch/_dynamo/variables/user_defined.py

```diff
@@ -489,6 +489,8 @@ def call_function(
         elif issubclass(self.value, enum.Enum) and len(args) == 1 and not kwargs:
             options = {"mutable_local": MutableLocal()}
             return variables.EnumVariable.create(self.value, args[0], options)
+        elif self.value is random.Random:
+            return RandomVariable(self.value, kwargs)
```

Comment:
This is incorrect. You will have to first create a random.Random object. For example, if you have a seed here. You should 
```
if len(args) == 1 and isinstance(args[0], ConstantVariable):
	seed = args[0].value

random_object = random.Random(seed)
return RandomVariable(random_object)
```